### PR TITLE
trim culprit

### PR DESCRIPTION
**Pull request by jalvz**

### CODE REVIEW

**Review by simitt — APPROVED** [review]

**Comment by simitt** [comment]
I cannot merge this unfortunately (no write access)

**Comment by jalvz** [comment]
thanks! me neither, but pushed directly here 5d4685cfc82e76c43095b70d9ab8642a5b65e53e

**Pull request closed without merging**
